src/lectern-client/schema-functions.ts

```diff
@@ -522,7 +522,7 @@ namespace validation {
       console.error(`failed running validation script ${field.name} for record: ${record}`);
       return {
         valid: false,
-        message: 'failed to run script validation, check script and the input',
+        message: `Failed to run script validation, check script and the input. Error: ${err}`,
```

Comment:
the problem is that data submitters [who have no control over these script] will see this message and it can mess the UI, do you think adding the error to the console.error before helps you ?

Reply:
I didn't realize this exact message also appeared on the UI. Adding it to the console error makes more sense.